x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/entity_analytics_management_page.cy.ts

```diff
@@ -147,10 +143,7 @@ describe(
 
         cy.get(RISK_SCORE_STATUS).should('not.exist');
 
-        updateRiskEngine();
-        updateRiskEngineConfirm();
-
-        cy.get(RISK_SCORE_STATUS).should('have.text', 'On');
```

Comment:
I believe we should keep the assertion outside the `updateRiskEngineConfirm();` because if not is kind of difficult to see what are we asserting :)

Reply:
@MadameSheema my idea was to make sure that upgrade was successful, I am not sure how we can do that without this line

Reply:
I see what you mean, in this case then I guess it is ok :)